src/App.js

```diff
@@ -87,12 +86,11 @@ class App extends Component {
     return (
       <Provider store={store}>
         <SearchProvider config={searchConfig}>
-        <BrowserRouter history={history} basename={baseURL}>
+        <BrowserRouter history={history} basename={packagejson.baseURL}>
           <ErrorBoundaryContainer>
             <NavBar app='atlas' />
             <Switch>
               <Route exact path="/" component={ImageDatasetListContainer} store={store} />
-              <Route exact path="/view" component={SpatialViewerContainer} store={store} />
```

Comment:
Removing spatial viewer as it isn't needed for the repository, and the vitessce package has a number of vulnerabilities associated with it.